src/efftester.ml

```diff
@@ -955,6 +989,7 @@ let createLit t =
   match t with
   | Unit   -> toTerm LitUnit
   | Int    -> toTerm (LitInt (Gen.generate1 small_int.gen))
+  | Float  -> toTerm (LitFloat (Gen.generate1 float.gen))
```

Comment:
Note: the `float` generator of QCheck is rather weak (see [source code](https://github.com/c-cube/qcheck/blob/b14291a/src/core/QCheck.ml#L118-L120)), generating a somewhat uniform random float. For a compiler tester you want to generate corner cases more often, such that `0., 1., -1.` to create optimizations opportunities, very small numbers (`Float.epsilon`) to test precision issues, negative zero to see some correctness bugs, `{min,max}_float`, `nan`, and positive/negative infinities.

We should experiment with a better generator locally, but if we have something nice we could also send it to the QCheck upstream.

Reply:
To clarify: I think that the present generator is fine for the current PR, we can experiment with stronger generators as followup work.